processor/processor.go

```diff
@@ -332,7 +332,7 @@ func setupCountAs() {
 			// See if we can identify based on language name which is the most
 			// reliable as the name should be unique
 			for name := range languageDatabase {
-				if strings.ToLower(name) == strings.ToLower(t[1]) {
+				if strings.EqualFold(name, t[1]) {
```

Comment:
Note for myself, remove this

See https://github.com/boyter/scc/pull/479#discussion_r1632882275

cc @apocelipes

Reply:
I removed in favor of #479